packages/components/psammead-media-player/README.md

```diff
@@ -19,18 +19,33 @@ This component to be used at any point on the page, specifically when a media pl
 | Argument  | Type                | Required | Default | Example         |
 |-----------|---------------------|----------|---------|-----------------|
 | `src` | string | Yes   | - | `http://foobar.com/embeddable_endpoint` |
+| `skin` | string | No   | `"classic"` | `"audio"` |
```

Comment:
```suggestion
| `skin` | string | No   | `'classic'` | `'audio'` |
```

I think we should stay consistent with single quotes like other components:
https://github.com/bbc/psammead/tree/latest/packages/components/psammead-play-button#props
https://github.com/bbc/psammead/tree/latest/packages/components/psammead-timestamp#props
https://github.com/bbc/psammead/tree/latest/packages/components/psammead-most-read#mostreaditem-props